nikola/cli.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+
+
+def parse_args(args=None):
+    parser = argparse.ArgumentParser()
+    add_arguments(parser)
+    options, args = parser.parse_known_args(args)
+    options = process_options(options) or options
+    return options, args
+
+
+def add_arguments(parser):
+    parser.add_argument('--conf', dest='conf_filename')
+    parser.add_argument('-D', '--define', dest='defines', action='append')
+    parser.add_argument('-o', '--output')
```

Comment:
To change the output folder once doesn’t make much sense (just copy an existing `output` directory somewhere), to change it regularly, you probably want a different `conf.py` file anyway.